src/index.css

```diff
@@ -11,3 +11,7 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+#multi-select-field > div > div > div{
+  height: 23px;
```

Comment:
there must be a way to not hard-code this value

Reply:
I looked in the react admin documentation, how to overwrite the style but doesn´t take effect. So I decided to look into MUI documentation and also didn´t work. That´s the reason why I decided to do in that way. But I'll keep searching how to modify through react-admin.